app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -260,6 +260,15 @@ def add_middleware_deployment(ems_ref, hash)
           :binary_content        => hash[:file]["file"].read,
           :resource_path         => ems_ref.to_s
         }
+        unless hash[:file]['server_groups'].nil?
+          # in case of deploying into server group the resource path should point to the domain controller
+          deployment_data[:server_groups] = hash[:file]['server_groups']
+          server_group_path = ::Hawkular::Inventory::CanonicalPath.parse(deployment_data[:resource_path])
+          server_group_path_hash = server_group_path.to_h
+          server_group_path_hash[:resource_ids] = [server_group_path_hash[:resource_ids].first]
```

Comment:
I actually want to drop all the elements except the first one.
`[1,2,3,4,5].slice!(1..-1)` does the opposite -> `[2, 3, 4, 5]`